src/components/Article.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Article = (
+  {
+    title,
+    text,
+    date,
+  },
+) => (
+  <>
```

Comment:
```suggestion
  <article>
```
Why would you use the fragment if there is more semantic tag?

Reply:
fixed